README.md

```diff
@@ -1,3 +1,75 @@
 # Procheck
 
-A project convention checker
+A project convention checker by checking file existence and word existence in a file, using a user-defined configuration. This allows users standardize their many projects using a simple, centralized configurations.
```

Comment:
Keep the desc simple : `A project convention checker`. But if you need to describe available features please create new section to describe available features.

Reply:
It should be fixed in the newest commit